packages/std/Cargo.toml

```diff
@@ -12,6 +12,14 @@ readme = "README.md"
 circle-ci = { repository = "CosmWasm/cosmwasm", branch = "master" }
 maintenance = { status = "actively-developed" }
 
+[features]
+# iterator allows us to iterate over all DB items in a given range
+# optional as some backends (like tries) don't support this
```

Comment:
What is tries backend?

Reply:
Ethereum, parity and some other projects use Patricia tries to store data. While cosmos uses an iavl tree.

Tries have some advantages, like predictable depth and the same hash regardless of insert order. While trees produce smaller proofs and allow iteration over the key space.

The reason merkle tries cannot iterate is that the keys are hashed before inserting to ensure they are all exactly 32 bytes long

Reply:
Great, thanks. Could you elaborate a bit more in the comment in code to persist that knowledge?

Reply:
done